lib/components/narrative/narrative-itineraries.js

```diff
@@ -256,6 +260,10 @@ class NarrativeItineraries extends Component {
 
     if (!activeSearch) return null
 
+    // render lists become divs if itinerary is expanded, to avoid rendering a list with list item
+    const ListContainer = itineraryIsExpanded ? 'div' : 'ul'
+    const ListItem = itineraryIsExpanded ? 'div' : 'li'
```

Comment:
Not needed per https://github.com/opentripplanner/otp-react-redux/pull/744#discussion_r1063525664.

Reply:
I think the same issue we pointed out in the above thread still applies - removing the outer <ul> means the batch-itin doesn't render as a list. Let me know if I'm misunderstanding!

Reply:
To preserve that, how about moving the `ListContainer` at line 399 to the block around line 429 (make it surround `itinerariesWithCo2.map(...)`.

Reply:
That works great, thanks so much Binh!